classes/Task/Update/UpdateFiles.php

```diff
@@ -97,17 +99,12 @@ public function run(): int
      * upgradeThisFile.
      *
      * @param mixed $orig The absolute path to the file from the upgrade archive
+     * @param string $file Relative file path from root
      *
      * @throws Exception
      */
-    public function upgradeThisFile($orig): bool
+    protected function upgradeThisFile($orig, string $file, ?string $symbolicLinkTarget): bool
```

Comment:
I'm wondering if we should have a dedicated method for symbolic links, because I see them being handled as special cases. This one would remain for the current management of basic files and folders.